YvesIraguha/CodeLab-ReactNative
# #166258436 Build Profile Screen

#### What does this PR do?
- Build a profile screen
#### Description of Task to be completed?
- Redirect a user to a profile screen. 
#### How should this be manually tested?
- Clone the repo and check out `ft-profile-screen-166258436 `
- Install all dependencies by `yarn`
- In project root directory run `yarn run test` to run tests. Tests should pass. 
- Run `yarn run start` to start the server 
- Follow the directions given, in the terminal. 
- You should see the list of engineers either in your simulator or your device.
- You should be able to see the profile screen when you click on one of the icons on the profile page
#### What are the relevant pivotal tracker stories?
#166258436

 #### Screenshots (if appropriate)
-Design 
<img width="267" alt="Screen Shot 2019-06-06 at 10 20 29" src="https://user-images.githubusercontent.com/34691070/59017806-c07bc000-8844-11e9-95dd-9bcbfd946417.png">

-Actual implementation 
<img width="365" alt="Screen Shot 2019-06-06 at 10 30 13" src="https://user-images.githubusercontent.com/34691070/59018417-23ba2200-8846-11e9-9ce0-808f886e3f05.png">
